lib/driver.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'csv_record'
+
+module RideShare
+    class Driver < CsvRecord
+      attr_reader :name, :vin, :status, :trips
+
+      def initialize(id:, name:, vin:, status: :AVAILABLE, trips: [])
+        super(id)
+
+        raise ArgumentError.new("Invalid VIN number.") unless vin.length == 17
+        raise ArgumentError.new("Invalid status") unless [:AVAILABLE, :UNAVAILABLE].include?(status)
```

Comment:
It's helpful to include the invalid arguments in your messages:

```suggestion
        raise ArgumentError.new("Invalid VIN number: #{vin}") unless vin.length == 17
        raise ArgumentError.new("Invalid status: #{status}") unless [:AVAILABLE, :UNAVAILABLE].include?(status)
```